e2e-tests/demand-backup-sharded/compare/statefulset_some-name-mongos.yml

```diff
@@ -191,3 +186,7 @@ spec:
             name: some-name-mongos
             optional: true
           name: config
+  updateStrategy:
+    rollingUpdate:
+      partition: 0
+    type: RollingUpdate
```

Comment:
I think `updateStrategy: SmartUpdate` is used in `demand-backup-sharded` test, but I still see `RollingUpdate` here.

Reply:
If `updateStrategy` is not specified, `updateStrategy` for all statefulsets is set to `RollingUpdate`